BranchMetrics/ios-branch-deep-linking-attribution
# [SDK-2095] Added new unit tests for main classes

## Reference
SDK-2095 -- iOS Test coverage - Add tests for Branch.m, BranchEvent, BranchUniversalObject, and BranchShareLink

## Summary
Add more tests to increase coverage, mostly in the main Branch.m class.

## Motivation
To increase test coverage percentage for main public methods.

## Type Of Change


## Testing Instructions

Run the new tests and ensure they are relevant and in the right locations.









cc @BranchMetrics/saas-sdk-devs for visibility.
